COPYRIGHT.txt

```diff
@@ -166,18 +166,18 @@ Comment: etcpak
 Copyright: 2013-2021, Bartosz Taudul
 License: BSD-3-clause
 
-Files: ./thirdparty/fonts/DroidSans*.ttf
-Comment: DroidSans font
+Files: ./thirdparty/fonts/DroidSans*.woff2
+Comment: DroidSans font, converted from .ttf
```

Comment:
I wouldn't bother specifying this in COPYRIGHT.txt, that doesn't sound like a change that warrants being mentioned in this context.

Reply:
OK, removed.